### ISSUE TITLE
server command to pause the game

### ISSUE THREAD

**Issue by hugomflavio** [longturn/freeciv21#2396]
**Is your feature request related to a problem? Please describe.**
I looked through [this webpage](https://freeciv.fandom.com/wiki/Server_commands) but couldn't find something that pauses the game.

**Describe the solution you'd like**
A command to freeze the game in its current state, available only in access level "hack." This should include freezing the time left in the turn. To be used when a conflict is being resolved that might require editing the current state of the game.

**Comment by jwrober** [comment]
FWIW, that is old. Our server manual is more comprehensive to fc21 https://longturn.readthedocs.io/en/latest/Manuals/Server/index.html

**Comment by hugomflavio** [comment]
Hah, that is pretty cool, thanks! I think we still a new command for this though, right? I had a quick look but couldn't find something that pauses the game.

**Comment by jwrober** [comment]
Yes agree, just wanted to point out that we have our own server manual now that get's all the current avail commands.
